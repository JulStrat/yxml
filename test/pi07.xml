<?xml version="1.0"?><?xml-stylesheet href="mystyle.css" type="text/css"?><a/>
