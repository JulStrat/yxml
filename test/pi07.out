
pistart xml-stylesheet
picontent href="mystyle.css" type="text/css"
piend
elemstart a
elemend
ok
